tests/unit/lms/services/oauth1_test.py

```diff
@@ -100,6 +58,17 @@ def test_sign(
                 "GET",
                 "Jg5MXVnexhzMDTv7IBUy3goIGqc=",
             ),
+            # https://lti.tools/oauth/ with query param
```

Comment:
The other test still sign to the same string which is expected.

Added a new test with query params.